gammapy/spectrum/flux_point.py

```diff
@@ -778,7 +786,11 @@ def compute_flux_point_sqrt_ts(self, fit, best_fit):
         ts = np.abs(stat_null - stat_best_fit)
         return np.sign(amplitude) * np.sqrt(ts)
 
-    def fit_point(self, model, energy_group, energy_ref, sqrt_ts_threshold=1):
+    def fit_point(self, model, energy_group, energy_ref, nsigma_ul_threshold=1.5, nsigma_ul=3.):
+        """
+        :param sqrt_ts_threshold: Significance for an energy bin for which an UL should be computed
```

Comment:
I'm not sure if this is 'allowed' in gammapy. Usually the docstring look like this

```
"""
Short decription

Long description

Parameters
--------------
name : type
    Description
name2 : type2
    Description 2
"""
```
See also [here](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)

Reply:
Yes, we're using the "Numpy docstring format", like Numpy, Astropy and most other scientific Python packages. The current docstring will raise a warning and then fail the docs CI build on travis-ci, i.e. this needs to be changed before we merge.
To test locally:
1. `python setup.py build_sphinx`
2. coffee
3. `make doc-show` or open `docs/_build/html/index.html` directly if `make doc-show` doesn't work.

Reply:
Indeed, this does not work so well, except for the coffee step;)
I have changed them to the standard docstring method